app/app.go

```diff
@@ -478,7 +575,9 @@ func New(
 	}
 
 	if upgradeInfo.Name == planName && !app.UpgradeKeeper.IsSkipHeight(upgradeInfo.Height) {
-		storeUpgrades := storetypes.StoreUpgrades{}
+		storeUpgrades := storetypes.StoreUpgrades{
+			Added: []string{icacontrollertypes.StoreKey, icahosttypes.StoreKey},
```

Comment:
the ica auth module doesn't need to be added?

Reply:
ica auth module does not access the store. Anyway, I've added it to the store upgrades just to be safe.

Reply:
and it doesn't have any params etc., so no need to initialize anything in the upgrade handler?
or should that 5minute timeout be a param?

Reply:
It doesn't have any params right now. But, it's a good idea to move default timeout duration to params. I'll do that in future PR.

Reply:
Added an option here: https://github.com/crypto-org-chain/chain-main/issues/731#issue-1171885779